.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+- name: Localisation
+  url: https://github.com/OpenRCT2/Localisation
+  about: For localisation issues, go to the localisation repository.
+- name: OpenRCT2 Discord Server
+  url: https://discord.gg/ZXZd8D8
+  about: Join our Discord server.
```

Comment:
How about "If you need help running or playing the game, join our Discord server"?

Reply:
Yes, that sounds more inviting. I went with "For help with running or playing the game, join our Discord server".